packages/eds-core-react/stories/playground/DataTable.tsx

```diff
@@ -112,7 +113,45 @@ const toCellValues = (data: Data[], columns: Column[]) =>
 
 const MenuButton = ({ row }: { row: string[] }) => {
   const [anchorEl, setAnchorEl] = useState<HTMLElement>()
-  const isOpen = Boolean(anchorEl)
+  const isElement = Boolean(anchorEl)
```

Comment:
Maybe keep this as `isOpen`? Looks confusing when you look at `Menu` and `open={isElement}`. Not sure If I understood the need for an additional `useState` 🤔